android-app/app/src/main/java/de/dhbw/ka/se/fibo/SplashActivity.java

```diff
@@ -0,0 +1,43 @@
+package de.dhbw.ka.se.fibo;
+
+import android.annotation.SuppressLint;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.util.Objects;
+
+@SuppressLint("CustomSplashScreen")
+// We need a custom splash screen because we want to support Android versions below 11
+public class SplashActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+
+
+        ActionBar supportActionBar = getSupportActionBar();
+        Objects.requireNonNull(supportActionBar);
+        supportActionBar.hide();
+
+        Thread welcomeThread = new Thread() {
+            @Override
+            public void run() {
+                try {
+                    super.run();
+                    sleep(2000);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                } finally {
+                    Intent i = new Intent(SplashActivity.this,
+                            MainActivity.class);
+                    startActivity(i);
+                    finish();
+                }
+            }
+        };
+        welcomeThread.start();
+    }
```

Comment:
we could outsource this to an event handler `onResume()` that checks whether we already initiated a thread and handles this.

Something like this:
```java
    @Override
    protected void onResume() {
        Log.v("FiBo","SplashActivity#onResume()");
        super.onResume();

        if (this.welcomeThread == null) {
            this.welcomeThread = new Thread() {
                @Override
                public void run() {
                    try {
                        super.run();
                        sleep(2000);
                    } catch (Exception e) {
                        e.printStackTrace();
                    } finally {
                        Intent i = new Intent(SplashActivity.this,
                                MainActivity.class);
                        startActivity(i);
                        finish();
                    }
                }
            };
            this.welcomeThread.start();
        }
    }
```